src/buffer/out/OutputCellIterator.cpp

```diff
@@ -97,14 +97,14 @@ OutputCellIterator::OutputCellIterator(const std::wstring_view utf16Text) :
 // Arguments:
 // - utf16Text - UTF-16 text range
 // - attribute - Color to apply over the entire range
-OutputCellIterator::OutputCellIterator(const std::wstring_view utf16Text, const TextAttribute attribute) :
+OutputCellIterator::OutputCellIterator(const std::wstring_view utf16Text, const TextAttribute& attribute, const size_t fillLimit) :

```

Comment:
wat

Reply:
I needed that so that I can write a unit test with surrogate characters.
The other tests only pass a single `wchar_t`...

Reply:
l o l